src/store/wallet.js

```diff
@@ -106,6 +114,15 @@ export default {
     setMnemonics(state, result) {
       Object.assign(state, { mnemonics: result })
     },
+    setNodeStatus(state, { status, eventType }) {
+      state.nodeStatus = {
+        node: status.node.address,
+        block: status.node.last_beacon.checkpoint.toString(),
+        network: null,
```

Comment:
Let's connect `network`,  as it is already working with my latest changes on my wallet PR.
```suggestion
        network: status.node.network,
```